src/sentry/models/integrations/organization_integration.py

```diff
@@ -56,3 +56,59 @@ def delete(self, *args, **kwds):
             for outbox in self.outboxes_for_update():
                 outbox.save()
             super().delete(*args, **kwds)
+
+    @staticmethod
+    def services_in(config: Mapping[str, Any]) -> List[PagerDutyServiceDict]:
+        return config.get("pagerduty_services", [])
+
+    def set_services(self, services: List[PagerDutyServiceDict]) -> None:
+        self.config["pagerduty_services"] = services
+
+    @staticmethod
+    def find_service(config: Mapping[str, Any], id: int | str) -> PagerDutyServiceDict | None:
+        try:
+            return next(
+                pds
+                for pds in OrganizationIntegration.services_in(config)
+                if str(pds["id"]) == str(id)
+            )
+        except StopIteration:
+            return None
+
+    def add_pagerduty_service(
+        self, integration_key: str, service_name: str
+    ) -> PagerDutyServiceDict:
+        from sentry.models import OrganizationIntegration
```

Comment:
Aren't you inside this model already?

Reply:
Correct, this is a copy paste fragment, will remove.